explorer/gql/gql.ts

```diff
@@ -48,7 +48,7 @@ const documents = {
     "\n  query ExtrinsicsSummary($first: Int!, $subspaceAccount: String) {\n    extrinsics: extrinsicsConnection(\n      orderBy: id_DESC\n      first: $first\n      where: { signer: { id_eq: $subspaceAccount } }\n    ) {\n      edges {\n        node {\n          id\n          success\n          block {\n            id\n            timestamp\n            height\n          }\n          name\n        }\n      }\n      totalCount\n    }\n  }\n": types.ExtrinsicsSummaryDocument,
     "\n  query StakingSummary($first: Int!, $subspaceAccount: String) {\n    operators: operatorsConnection(\n      orderBy: id_ASC\n      first: $first\n      where: { operatorOwner_eq: $subspaceAccount }\n    ) {\n      edges {\n        node {\n          id\n          operatorOwner\n          currentDomainId\n          currentTotalStake\n          totalShares\n        }\n      }\n      totalCount\n    }\n    nominators: nominatorsConnection(\n      orderBy: id_ASC\n      first: $first\n      where: { account: { id_eq: $subspaceAccount } }\n    ) {\n      edges {\n        node {\n          id\n          shares\n          account {\n            id\n          }\n          operator {\n            id\n            operatorOwner\n            currentDomainId\n            currentTotalStake\n            totalShares\n          }\n        }\n      }\n      totalCount\n    }\n  }\n": types.StakingSummaryDocument,
-    "\n  query CheckRole($subspaceAccount: String!) {\n    isFarmer: rewardEvents(\n      where: { name_eq: \"Rewards.VoteReward\", account: { id_eq: $subspaceAccount } }\n      limit: 1\n    ) {\n      account {\n        id\n      }\n    }\n  }\n": types.CheckRoleDocument,
+    "\n  query CheckRole($subspaceAccount: String!) {\n    farmer: rewardEvents(\n      where: { name_eq: \"Rewards.VoteReward\", account: { id_eq: $subspaceAccount } }\n      limit: 1\n    ) {\n      account {\n        id\n      }\n    }\n    operator: operatorsConnection(\n      first: 1\n      where: { operatorOwner_eq: $subspaceAccount }\n      orderBy: id_ASC\n    ) {\n      totalCount\n    }\n    nominator: nominatorsConnection(\n      first: 1\n      where: { account: { id_eq: $subspaceAccount } }\n      orderBy: id_ASC\n    ) {\n      totalCount\n    }\n  }\n": types.CheckRoleDocument,
```

Comment:
Nice and clean!

Reply:
This file is automatically generated when running `yarn codegen` 👍